evalml/automl/automl_search.py

```diff
@@ -667,6 +673,18 @@ def __init__(
             pipeline_params=parameters,
             custom_hyperparameters=custom_hyperparameters,
         )
+        self._deactivate_logger()
+
+    def _activate_logger(self):
+        if self.verbose:
+            self.logger = get_logger(__name__)
+        else:
+            self.logger = logging.getLogger(__name__)
+
+    def _deactivate_logger(self):
+        if self.verbose and len(self.logger.handlers):
+            self.logger.handlers.pop()
+            self.logger.setLevel(logging.WARNING)
```

Comment:
Remind me once more why we reset the logger level to WARNING here?

Suppose a user has an application where, somewhere down in the stack, they run `automl=AutoMLSearch(..., verbose=True); automl.search()` and then delete the automl object. And let's say first they set the global log level to INFO. By setting the log level here, is it possible this line would reset the global log level to WARNING, overriding their original setting?